js/base-chat.js

```diff
@@ -713,6 +725,29 @@ function baseChat () {
 		if (!params[2]) {
 			d20plus.chat.resetSendMyself();
 		}
+		const macroJS = d20plus.cfg.getOrDefault("chat", "executeJSMacro");
+
+		if (macroJS !== "none") {
+			const template = /#(?<macroid>[^ ^#]+)/g;
+			params[0] = params[0].replace(template, (...string) => {
```

Comment:
not-so-nit: `string` -> `match`

Reply:
done. I've got a lot of replacements to make in the code that hasn't got to any PR yet...